src/goPath.ts

```diff
@@ -27,6 +28,13 @@ export function getBinPath(binname) {
 	return path.join(process.env["GOPATH"], "bin", binname);
```

Comment:
What would you suggest instead?  Perhaps just return `binname` and rely on finding it in `PATH`?